wg-charter.html

```diff
@@ -0,0 +1,406 @@
+              <li>Reference implementations of the group's deliverables</li>
+              <li>Use case and requirement documents</li>
+              <li>Explainers, primers and best-practice documents</li>
+            </ul>
+        </div>
+      </section>
+
+
+      <section id="success-criteria">
+        <h2>Success Criteria</h2>
+        <p>In order to advance to  <a href="https://www.w3.org/2018/Process-20180201/#RecsCR" title="Proposed Recommendation">Proposed Recommendation</a>, each specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations of each feature defined in the specification</a>.</p>
+        <p>Each specification should contain a section detailing any <a href='https://www.w3.org/TR/security-privacy-questionnaire/'>known security</a>, fingerprinting, and privacy implications, and suggested <a href='https://w3c.github.io/perf-security-privacy/'>mitigation strategies</a> for implementers, web authors, and end users. The group should not publish a specification if acceptable mitigation strategies cannot be found.</p>
+        <p>Each specification should contain a section describing known impacts on accessibility to users with disabilities, ways the specification features address them, and recommendations for minimizing accessibility problems in implementation.</p>
+        <p>Normative specification changes are generally expected to have a corresponding set of tests, either in the form of new tests or modifications to existing tests, or must include the rationale for why test updates are not required for the proposed update.</p>
+      </section>
+
+
+      <section id="coordination">
+        <h2>Coordination</h2>
+        <p>For all specifications, this Working Group will seek <a href="https://www.w3.org/Guide/Charter.html#horizontal-review">horizontal review</a> for accessibility, internationalization, performance, privacy, and security with the relevant Working and Interest Groups, and with the <a href="https://www.w3.org/2001/tag/" title="Technical Architecture Group">TAG</a>. Invitation for review must be issued during each major standards-track document transition, including <a href="https://www.w3.org/2018/Process-20180201/#RecsWD" title="First Public Working Draft">FPWD</a> and <a href="https://www.w3.org/2018/Process-20180201/#RecsCR" title="Candidate Recommendation">CR</a>, and should be issued when major changes occur in a specification.</p>
+
+        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/2018/Process-20180201/#WGCharter">W3C Process Document</a>:</p>
+
+        <div>
+          <h3 id="w3c-coordination">W3C Groups</h3>
+          <dl>
+            <dt><a href="https://www.w3.org/2001/tag/">Technical Architecture Group (TAG)</a></dt>
+        <dd>The WG may ask the Technical Architecture Group to review some of its specifications.</dd>
```

Comment:
This should be "will ask" instead of "may ask" to match what's said in 'coordination":

> For all specifications, this Working Group will seek horizontal review ... , and with the TAG

Reply:
That's a good point, but I believe the fix is to drop the TAG from this list, because there's simply no reason to include it in the first place. That list is meant to convey "additional" groups with which the GPU for the Web WG intends to exchange on top of those that will de facto run reviews on the specs such as the TAG.